source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRTextInputHandler.h

```diff
@@ -22,9 +22,9 @@
 
 @interface ACRNumberInputHandler : ACRTextInputHandler
 
-@property NSInteger min;
+@property NSNumber *min;
```

Comment:
why not CGFloat?

Reply:
Looks like CGFloat is typedef for `float` on 32-bit arch. Wouldn't that mean we truncate the value coming from the shared model?

Reply:
I could use a plain double, though

Reply:
double is also fine.